projects/plugins/jetpack/modules/masterbar/admin-menu/class-base-admin-menu.php

```diff
@@ -225,16 +222,23 @@ public function enqueue_scripts() {
 	}
 
 	/**
-	 * Remove submenu items from given menu slug.
+	 * Adds the given menu item in the specified position.
 	 *
-	 * @param string $slug Menu slug.
+	 * @param array $item The menu item to add.
+	 * @param int   $position The position in the menu order this item should appear.
 	 */
-	public function remove_submenus( $slug ) {
```

Comment:
Interesting, this one was not used at all?

Reply:
Nope, seems we forgot to remove it at some point